### PR TITLE
doc: add v0.8 release note

### PR DESCRIPTION
**Pull request by wenlingz**
v0.8 release note
Signed-off-by: wenlingz <wenling.zhang@intel.com>

### CODE REVIEW

File: doc/release_notes_0.8.rst [hunk 1]
```diff
@@ -0,0 +1,389 @@
...
+instructions in the :ref:`getting-started-apl-nuc`.
+
+Version 0.8 new features
+************************
+
+GPIO virtualization
+=========================
+
+GPIO virtualization is supported as para-virtualization based on Virtual 
+I/O Device(VIRTIO) specification. The GPIO consumers of the Front-end are
```
Comment (gvancuts) [inline doc/release_notes_0.8.rst]:
```suggestion
I/O Device (VIRTIO) specification. The GPIO consumers of the Front-end are
```

File: doc/release_notes_0.8.rst [hunk 2]
```diff
@@ -0,0 +1,389 @@
...
+able to set or get GPIO values, directions and configuration via one virtual 
+GPIO controller. In the Back-end, GPIO command line in launch script 
+could be modified to map native GPIO to UOS.
+
+Enable QoS based on runC container
+==================================
+ACRN supports Device-Model QoS based on runC container to control the SOS
+resources(CPU/Storage/MEM/NET) by modifying the runC configuration file.
```
Comment (gvancuts) [inline doc/release_notes_0.8.rst]:
```suggestion
resources (CPU/Storage/MEM/NET) by modifying the runC configuration file.
```

File: doc/release_notes_0.8.rst [hunk 3]
```diff
@@ -0,0 +1,389 @@
...
+ACRN supports shutdown the RTVM from the internal of the VM, since RTVM 
+is the kind of VM that can’t be interfered by SOS at runtime and could 
+only be powered of by itself from internal. All other poweroff request 
+from external will be rejected directly to avoid the interference.
+
+Document updates
+================
+Several new documents have been added in this release, including:
+
+* Zephyr as Guest
```
Comment (gvancuts) [inline doc/release_notes_0.8.rst]: Could you add a link to all those new documents?

**Comment by dbkinder** [comment]
@NanlinXie I addressed @gvancuts  and my comments and submitted a patch directly to @wenlingz PR (eliminating another loop for him to fix and us to review).  This can be merged when you see this in the morning.

File: doc/release_notes_0.8.rst [hunk 4]
```diff
@@ -0,0 +1,391 @@
...
+
+- :acrn-issue:`923` - GPU Mediator shall be compatible with operation of graphics safety watchdog
+- :acrn-issue:`1409` - Add support for profiling [sep/socwatch tools]
+- :acrn-issue:`1568` - Implement PCI emulation functionality in HV for UOS passthrough devices and SOS MSI/MSI-X remapping
+- :acrn-issue:`1867` - vMSR code reshuffle
+- :acrn-issue:`2020` - DM: Enable QoS in ACRN, based on runC container
+- :acrn-issue:`2512` - GPIO virtualization
+- :acrn-issue:`2605` - hv: search additional argument when parsing seed from ABL(apl_sdc_stable)
+- :acrn-issue:`2611` - hv: search additional argument when parsing seed from ABL.
+- :acrn-issue:`2662` - OVMF release v0.7
```
Comment (gvancuts) [inline doc/release_notes_0.8.rst]: It seems a bit odd we're calling out v0.7 as a new feature while we have actually merged v0.8 already in this release?

File: doc/release_notes_0.8.rst [hunk 5]
```diff
@@ -0,0 +1,391 @@
...
+=========================
+
+GPIO virtualization is supported as para-virtualization based on the
+Virtual I/O Device (VIRTIO) specification. The GPIO consumers of the
+Front-end are able to set or get GPIO values, directions, and
+configuration via one virtual GPIO controller. In the Back-end, the GPIO
+command line in the launch script can be modified to map native GPIO to
+UOS.
+
+Enable QoS based on runC container
```
Comment (gvancuts) [inline doc/release_notes_0.8.rst]: Is there any good pointer we can use here for those who want to learn more about this feature?

File: doc/release_notes_0.8.rst [hunk 6]
```diff
@@ -0,0 +1,392 @@
...
+New Features Details
+********************
+
+- :acrn-issue:`923` - GPU Mediator shall be compatible with operation of graphics safety watchdog
+- :acrn-issue:`1409` - Add support for profiling [sep/socwatch tools]
+- :acrn-issue:`1568` - Implement PCI emulation functionality in HV for UOS passthrough devices and SOS MSI/MSI-X remapping
+- :acrn-issue:`1867` - vMSR code reshuffle
+- :acrn-issue:`2020` - DM: Enable QoS in ACRN, based on runC container
+- :acrn-issue:`2512` - GPIO virtualization
+- :acrn-issue:`2605` - hv: search additional argument when parsing seed from ABL(apl_sdc_stable)
```
Comment (dbkinder) [inline doc/release_notes_0.8.rst]: @NanlinXie recommends deleting this issue from the list

File: doc/release_notes_0.8.rst (hunk 4 above)
Comment (NanlinXie) [reply doc/release_notes_0.8.rst]: We have another git issue to trace v0.8, will update it soon.

File: doc/release_notes_0.8.rst (hunk 6 above)
Comment (wenlingz) [reply doc/release_notes_0.8.rst]: OK

File: doc/release_notes_0.8.rst (hunk 4 above)
Comment (wenlingz) [reply doc/release_notes_0.8.rst]: Will modify it

File: doc/release_notes_0.8.rst (hunk 5 above)
Comment (NanlinXie) [reply doc/release_notes_0.8.rst]: Current not yet. I will ask developer to provide this guide in next release.

**Pull request merged**